### DISCUSSION TITLE
OPA keep-alive settings

### DISCUSSION THREAD

**Discussion by somsom13** [open-policy-agent/community#623] [OPA and Rego]
Hi, I have a question about OPA's keep-alive setting.

1. Does OPA support keep-alive connections? 
2. If supported, I was wondering what the keep-alive timeout default is set, and if there is a way for the user to set the timeout.

Thanks for your support!

**Comment by anderseknert** [comment]
Hi there! Yes, the server supports keep-alive connections, and from looking at the code the ttl is hard-coded to 3 minutes. If there are good reasons to make that configurable, I'd suggest opening an issue :)

 https://github.com/open-policy-agent/opa/blob/db0a80b4adb57fd2ca48a513af630832f48f6159/server/server.go#L476-L490
